### PR TITLE
NEXUS-35780: rollbacked change to get a specific nexus-pro-image version.

### PR DESCRIPTION
**Pull request by CodingLeoo**

### CODE REVIEW

**Review by Blacktiger — APPROVED** [review]

File: maven-plugin/src/test/resources/staging/staging.properties [hunk 1]
```diff
@@ -1,16 +1,3 @@
-#
```
Comment (mpiggott) [inline maven-plugin/src/test/resources/staging/staging.properties:1]: The change here seems unnecessary

File: testsuite/pom.xml [hunk 2]
```diff
@@ -28,6 +28,7 @@
   <artifactId>nxrm3-maven-plugin-testsuite</artifactId>
   
   <properties>
+    <nexus3.version>3.41.1-01</nexus3.version>
```
Comment (mpiggott) [inline testsuite/pom.xml:31]: Haven't we released 3.42? Why 3.41.1?

What will ensure we update this upon releases?

Comment (CodingLeoo) [reply testsuite/pom.xml:31]: Checking RSO , seems like there's no 3.42.0 for the nxrm-pro-image package : 
Screen Shot 2022-10-25 at 4 20 29 PM

File: maven-plugin/src/test/resources/staging/staging.properties (hunk 1 above)
Comment (CodingLeoo) [reply maven-plugin/src/test/resources/staging/staging.properties]: yep . rollbacked

**Pull request merged**